From 4f64069b498e77698d4ac5ac3de272aba7bd8ea1 Mon Sep 17 00:00:00 2001
From: Kurt Kiefer <kekiefer@gmail.con>
Date: Fri, 3 Mar 2023 15:55:39 -0800
Subject: [PATCH] Get CMAKE_BUILD_DIR from environment

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index e78d098..5cbd611 100644
--- a/setup.py
+++ b/setup.py
@@ -22,7 +22,7 @@ import setuptools.command.develop
 TOP_DIR = os.path.realpath(os.path.dirname(__file__))
 SRC_DIR = os.path.join(TOP_DIR, "onnx")
 TP_DIR = os.path.join(TOP_DIR, "third_party")
-CMAKE_BUILD_DIR = os.path.join(TOP_DIR, ".setuptools-cmake-build")
+CMAKE_BUILD_DIR = os.getenv("CMAKE_BUILD_DIR", os.path.join(TOP_DIR, ".setuptools-cmake-build"))
 PACKAGE_NAME = "onnx"
 
 WINDOWS = os.name == "nt"
