From a4fb17699dbb255bc895de5f3742932ab691e951 Mon Sep 17 00:00:00 2001
From: Kurt Kiefer <kekiefer@gmail.com>
Date: Wed, 8 Apr 2026 16:33:23 -0700
Subject: [PATCH] Remove unsupported pkg_resources references

Upstream-Status: Pending
---
 setup.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index 6088f1d..af62e9e 100644
--- a/setup.py
+++ b/setup.py
@@ -10,7 +10,6 @@ OmegaConf setup
 """
 import pathlib
 
-import pkg_resources
 import setuptools
 
 from build_helpers.build_helpers import (
@@ -24,8 +23,9 @@ from build_helpers.build_helpers import (
 
 with pathlib.Path("requirements/base.txt").open() as requirements_txt:
     install_requires = [
-        str(requirement)
-        for requirement in pkg_resources.parse_requirements(requirements_txt)
+        line.strip()
+        for line in requirements_txt
+        if line.strip() and not line.startswith("#")
     ]
 
 
-- 
2.34.1

