From 432170c6b427f76a60f8a23f5565f92e3596faa5 Mon Sep 17 00:00:00 2001
From: Kurt Kiefer <kurt.kiefer@arthrex.com>
Date: Wed, 27 Jul 2022 10:07:28 -0700
Subject: [PATCH] cuda: honor CUDAFLAGS for detection during
 enable_language(CUDA)

Signed-off-by: Kurt Kiefer <kurt.kiefer@arthrex.com>
---
 Modules/CMakeDetermineCUDACompiler.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Modules/CMakeDetermineCUDACompiler.cmake b/Modules/CMakeDetermineCUDACompiler.cmake
index d06315e4..58c669f2 100644
--- a/Modules/CMakeDetermineCUDACompiler.cmake
+++ b/Modules/CMakeDetermineCUDACompiler.cmake
@@ -270,6 +270,8 @@ if(NOT CMAKE_CUDA_COMPILER_ID_RUN)
     set(CMAKE_CUDA_COMPILER_ID_REQUIRE_SUCCESS ON)
   endif()
 
+  string(APPEND nvcc_test_flags " $ENV{CUDAFLAGS}")
+
   if(CMAKE_CUDA_COMPILER_ID STREQUAL "Clang")
     if(NOT CMAKE_CUDA_ARCHITECTURES)
       # Clang doesn't automatically select an architecture supported by the SDK.
