From 56d38267b496d12b156228aa05d1e6b45b1096d8 Mon Sep 17 00:00:00 2001
From: Kurt Kiefer <kurt.kiefer@arthrex.com>
Date: Tue, 16 Aug 2022 22:48:01 -0700
Subject: [PATCH] Fix splitting NVCC flags

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 543194515..c59a24a21 100644
--- a/setup.py
+++ b/setup.py
@@ -212,7 +212,7 @@ def get_extensions():
             if nvcc_flags == "":
                 nvcc_flags = []
             else:
-                nvcc_flags = nvcc_flags.split(" ")
+                nvcc_flags = nvcc_flags.split()
         else:
             define_macros += [("WITH_HIP", None)]
             nvcc_flags = []
