From c9b70f1ca6461dea5834db6d467df52922a2d17f Mon Sep 17 00:00:00 2001
From: Kurt Kiefer <kurt.kiefer@arthrex.com>
Date: Mon, 22 Aug 2022 15:54:16 -0700
Subject: [PATCH] cpp_extension: bypass cuda version checks

---
 torch/utils/cpp_extension.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/torch/utils/cpp_extension.py b/torch/utils/cpp_extension.py
index 57d870a3..09daead4 100644
--- a/torch/utils/cpp_extension.py
+++ b/torch/utils/cpp_extension.py
@@ -806,7 +806,7 @@ class BuildExtension(build_ext, object):
             if cuda_version is not None:
                 cuda_str_version = cuda_version.group(1)
                 cuda_ver = packaging.version.parse(cuda_str_version)
-                torch_cuda_version = packaging.version.parse(torch.version.cuda)
+                torch_cuda_version = cuda_ver
                 if cuda_ver != torch_cuda_version:
                     # major/minor attributes are only available in setuptools>=49.6.0
                     if getattr(cuda_ver, "major", float("nan")) != getattr(torch_cuda_version, "major", float("nan")):
