From 077c3734f737a149c7af4f2cab57e41e1a3839c6 Mon Sep 17 00:00:00 2001
From: Kurt Kiefer <kurt.kiefer@arthrex.com>
Date: Wed, 22 Feb 2023 21:53:27 -0800
Subject: [PATCH] Use external onnx build

---
 CMakeLists.txt | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c015bd9..74be642 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -66,14 +66,9 @@ else()
   set(PROTOBUF_LIB "protobuf::libprotobuf")
 endif()
 
-if(NOT TARGET onnx_proto)
-  # Note: This avoids libprotobuf.so complaining about name collisions at runtime
-  if(NOT ONNX_NAMESPACE)
-    set(ONNX_NAMESPACE "onnx2trt_onnx")
-  endif()
-  add_definitions("-DONNX_NAMESPACE=${ONNX_NAMESPACE}")
-  add_subdirectory(third_party/onnx EXCLUDE_FROM_ALL)
-endif()
+add_definitions("-DONNX_NAMESPACE=onnx")
+add_definitions("-DONNX_ML=ON")
+add_definitions("-DONNXIFI_ENABLE_EXT=ON")
 
 # CUDA
 if (NOT CUDA_TOOLKIT_ROOT_DIR)
