From 70af3a7fb67e5394ca544f78c0fc815db164af20 Mon Sep 17 00:00:00 2001
From: Kurt Kiefer <kekiefer@gmail.con>
Date: Tue, 7 Mar 2023 09:50:50 -0800
Subject: [PATCH] Link to external onnx

---
 CMakeLists.txt | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c015bd9..5493db5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -66,14 +66,7 @@ else()
   set(PROTOBUF_LIB "protobuf::libprotobuf")
 endif()
 
-if(NOT TARGET onnx_proto)
-  # Note: This avoids libprotobuf.so complaining about name collisions at runtime
-  if(NOT ONNX_NAMESPACE)
-    set(ONNX_NAMESPACE "onnx2trt_onnx")
-  endif()
-  add_definitions("-DONNX_NAMESPACE=${ONNX_NAMESPACE}")
-  add_subdirectory(third_party/onnx EXCLUDE_FROM_ALL)
-endif()
+FIND_PACKAGE(ONNX REQUIRED)
 
 # CUDA
 if (NOT CUDA_TOOLKIT_ROOT_DIR)
